packages/core/test/integration/command-test-runner/CommandTestRunner.it.spec.ts

```diff
@@ -13,7 +13,7 @@ describe(`${CommandTestRunner.name} integration`, () => {
 
   it('should report test as successful', async () => {
     const sut = createSut();
-    const result = await sut.run();
+    const result = await sut.run({ timeout: 100 });
```

Comment:
Writing it like this makes it seem that `timeout` is somehow significant. I took me a couple of seconds before I figured out you only need it here to satisfy the TypeScript compiler.

Please define a `const UNUSED_RUN_OPTIONS: RunOptions =  { timeout: 100 }` and use that instead of `{ timeout: 100 }`

Reply:
Yup, definitely cleaner.